app/models/warehouse/project.rb

```diff
@@ -10,5 +10,25 @@ class Project < Base
 
     belongs_to :organization, class_name: Warehouse::Organization.name, primary_key: [:OrganizationID, :data_source_id], foreign_key: [:OrganizationID, :data_source_id], inverse_of: :projects
 
+    def name
+      self.ProjectName
```

Comment:
I wonder if this should enforce returning "confidential project" if the project is confidential.  I can see a scenario where someone makes the attachment to a program and then the project is later flagged as confidential.